.travis.yml

```diff
@@ -13,14 +13,12 @@ python:
   - 2.7
 services:
   - redis-server
-before_script:
-  - mkdir imgee/static/test_uploads
 script:
-  - nosetests --with-coverage
+  - ./runtests.sh
 install:
   - sudo apt-get -qq install zlib1g-dev libfreetype6-dev liblcms1-dev libwebp-dev libjpeg-dev libpng-dev libfreetype6-dev libtiff4-dev librsvg2-dev ghostscript imagemagick pandoc
```

Comment:
Umm. It's not been removed. They are there. https://github.com/hasgeek/imgee/blob/e7f46fae57543ef3b1c77c17f3b190d60a53ea22/.travis.yml#L21